inst/templates/ad_advs.R

```diff
@@ -111,9 +112,9 @@ height_for_age <- who_lgth_ht_for_age_boys %>%
 # For weight-based indicators we keep WHO and CDC separate due to the WHO
 # adjustment (restricted application of the LMS method) needed for the
 # anthropometric indicator derivations
-data(who_wt_for_age_boys)
-data(who_wt_for_age_girls)
-data(cdc_wtage)
+who_wt_for_age_boys <- admiralpeds::who_lgth_ht_for_age_boys
+who_wt_for_age_girls <- admiralpeds::who_lgth_ht_for_age_girls
```

Comment:
Didn't you reverse the calls at lines 75 with lines 116 ?
e.g. should be `who_wt_for_age_girls <- admiralpeds::who_wt_for_age_girls`